src/commands/create.js

```diff
@@ -162,6 +162,7 @@ class CreateCommand extends ApifyCommand {
 
         if (dependenciesInstalled) {
             outputs.success(`Actor '${actorName}' was created. To run it, run "cd ${actorName}" and "apify run".`);
+            outputs.success('To push your actor to the Apify platform, run "apify push".');
```

Comment:
I'd probably go with something more generic, we don't want to use the Apify platform wording anymore (it should be console now).

@TheoVasilis The idea is to promote running Actors "in the cloud" - in the Apify console more and we could also add something with monetization. Users choose to install the Actor template locally on their computer and we want to display this message after the installation.

Can you please help us to figure this out?

Reply:
@jbartadev I think I need more context. Adding something about monetization here would seem out of place as far as I can tell from this. You can easily change this to 'To run your Actor in the cloud, run "apify push".' But not sure where to add monetization. It seems like a separate thing so it might need to go somewhere else.

Reply:
@TheoVasilis Yeah, sorry. What we had in mind was something like: 

`Do you want to run your code in the cloud and possibly publish it and earn passive income from it? Run "apify push" and deploy your code to Apify Console.`

But I guess you're right, it feels to long and maybe this isn't the right place for the monetization part. I'm not sure

Reply:
@jbartadev I'd put monetization somewhere else. So here you can have: 'To run your code in the cloud, run "apify push" and deploy your code to Apify Console. Another section could be 'Monetize your code': 'Do you want to publish your Actor and earn passive income? Deploy your code to Apify Console. You can do that by running "apify push". Not sure if that's clear enough.

Reply:
@TheoVasilis Thanks, in that case, I'd go just for

```suggestion
            outputs.success('To run your code in the cloud, run "apify push" and deploy your code to Apify Console.');
```